### ISSUE TITLE
All of my 900+ saves are gone!

### ISSUE THREAD

**Issue by Knockonwoodathome** [OpenEmu/OpenEmu#3070]
I play Monster Hunter Freedom Unite on my MacBook Air mid-2012 edition. Today, I was trying to repel a Lao-Shan-Lung, and my computer froze. I was having a problem with clicking on the game earlier, but I had fixed it. This time, I couldn't even log into my computer after I shut it off, so I turned it off and then on. When I came back to play, it asked me to select a library, so I created a new one. Next thing I know, all of my 950-something saves were missing, and all my data reset. PLEASE HELP! Sidenote: Upon restarting, I got a message saying "Chose OpenEmu Library: OpenEMu needs a library to continue. You may choose an existing OpenEmu library or create a new one. I selected create new, and lost all my data. Can I reverse this?

**Comment by clobber** [comment]
Stop posting in unrelated closed issues - that's annoying.

Maybe provide useful details about your setup such as OS X and OpenEmu versions?

**Comment by Knockonwoodathome** [comment]
Computer: MacBook Air (13-inch, Mid 2012)
            Processor: 1.8 GHz Intel Core i5
            Memory: 4 GB 1600 MHz DDR3
            Graphics: Intel HD Graphics 4000 1536 MB
            Serial Number: C02JKRATDRVC
            OpenEmu Version: Most Recent Version? I don't know how to find that information

**Comment by Knockonwoodathome** [comment]
Let me rephrase the question: How does OpenEmu store files on a Mac, and would creating a new library for these files delete the old libraries?

**Comment by clobber** [comment]
>OpenEmu Version: Most Recent Version? I don't know how to find that information

OpenEmu > About OpenEmu <--- the version is displayed in that dialog.

>How does OpenEmu store files on a Mac

Depends what you mean. Physical files are stored in various folders in `~/Library/Application Support OpenEmu`, while various metadata about the game library is kept in a datastore in `~/Library/Application Support OpenEmu/Game Library`, unless you changed the default location in OpenEmu Library Preferences.

>would creating a new library for these files delete the old libraries

It depends if you overwrote the old library location or created one in a new location. I don't know which files you are trying to recover, in-game mem card saves or savestates, and without detailed steps on how the problem occurred or a way to replicate it, I don't think I can help you much.

**Comment by gingerbeardman** [comment]
Do you not have a Time Machine backup to restore from?
